app/src/main/java/org/jellyfin/androidtv/data/service/BackgroundService.kt

```diff
@@ -40,7 +33,7 @@ import java.util.concurrent.ExecutionException
 class BackgroundService(
 	private val context: Context,
 	private val apiClient: ApiClient,
-	private val userPreferences: UserPreferences
+	private val userPreferences: UserPreferences,
```

Comment:
You committed this comma. Is that the new style for Kotlin or just preference? Did you mean to? Nothing follows.

Reply:
Kotlin 1.4 added trailing comma support. I'm slowly adding it to files I touch (formatter does it for me :p)

Reply:
Ok that's what I assumed. LGTM!